apps/staking/app/register/[nodeId]/Registration.tsx

```diff
@@ -251,13 +256,21 @@ function RegistrationProvider({
     (contract.status === CONTRIBUTION_CONTRACT_STATUS.WaitForOperatorContrib ||
       contract.status === CONTRIBUTION_CONTRACT_STATUS.OpenForPublicContrib);
 
-  const [tab, setTab] = useState<REG_TAB>(
-    addedBlsKeys?.has(blsKey)
+  const [_tab, setTab] = useState<REG_TAB>(
+    networkBlsKeys?.has(blsKey)
       ? REG_TAB.ALREADY_REGISTERED_RUNNING
       : alreadyRegisteredMulti
         ? REG_TAB.ALREADY_REGISTERED_MULTI
         : (queryParamStartTab ?? REG_TAB.START)
   );
+
+  const tab = useMemo(() => {
+    if (isVestingMode && !allowedVestingRegistrationTabs.includes(_tab)) {
```

Comment:
I assume this is defined in a later PR because it is not referenced anywhere?

Reply:
its defined above, but set to false until the next PR

Reply:
Defined in #46